src/main/java/core/basesyntax/strategy/handler/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImp;
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    private FruitDao dataBase;
+
+    public PurchaseOperationHandler() {
+        this.dataBase = new FruitDaoImp();
+    }
```

Comment:
Inject through the constructor means you should pass your dao object in the constructor arguments